interface-impl-amf/src/main/java/org/mule/amf/impl/parser/rule/ApiValidationResultImpl.java

```diff
@@ -35,9 +35,13 @@ public ApiValidationResultImpl(amf.client.validate.ValidationResult validationRe
 
   @Override
   public String getMessage() {
-    return buildErrorMessage(validationResult.message(),
-                             validationResult.location().orElse(""),
-                             validationResult.position().start());
+    String formattedErrorMessage =   format(validationResult.message());
+    Optional <String> location = validationResult.location();
+    Range positionRange = validationResult.position();
+    if(!location.empty().isPresent() && !positionRange.start().isZero()) {
```

Comment:
```location.empty()``` returns an empty optional, so ```!location.empty().isPresent()``` is always true. Get rid of ```empty()``` and the negation, leave just ```location.isPresent()``` to check if there is value for the optional